### DISCUSSION TITLE
Sprinkler lifespan?

### DISCUSSION THREAD

**Discussion by jmmullins** [jeremyckahn/farmhand#436] [Q&A] [answered]
Are sprinklers ever "lost" to anything? (storms, crows, rampant cows, malfunction, etc) 

Wondering if they'd be considered a "permanent" item until removed, or if you may have to periodically replace them like the scarecrows?

**Comment by jeremyckahn** [comment, marked as answer]
Currently Sprinklers don't get destroyed randomly like Scarecrows do from storms. Do you think that should be added in? Personally I like having them be durable items that don't wear out (such as by rusting), but I'm open to other opinions about that. Let me know what you think!

**Comment by jmmullins** [comment]
This question was the precursor to a request I'm going to make, just wanted to first confirm that sprinklers aren't randomly destroyed or need to be replaced over time. Thanks!
